dockstore-cli-integration-testing/src/test/java/io/dockstore/client/cli/WorkflowIT.java

```diff
@@ -352,10 +352,6 @@ public void cwlVersion11() {
             .filter(version -> "master".equalsIgnoreCase(version.getName())).findFirst();
         assertTrue(optionalWorkflowVersion.isPresent());
         WorkflowVersion workflowVersion = optionalWorkflowVersion.get();
-        List<SourceFile> sourceFiles = workflowVersion.getSourceFiles();
```

Comment:
This may still be in progress, but there's a new endpoint that needs to be called where you need tol validate the sourcefiles.

Reply:
Richard Hansen has fixed this